cluster-autoscaler/utils/drain/drain.go

```diff
@@ -33,6 +33,12 @@ const (
 	PodDeletionTimeout = 12 * time.Minute
 )
 
+const (
+	// PodSafeToEvictKey - annotation that ignores constraints to evict a pod like not being replicated, being on
+	// kube-system namespace, having a Disruption Budget (PDB) or having a local storage.
```

Comment:
Technically the comment is not true - this does not skip a PDB check (which is done in simulator/drain.go and not here). I don't think we should skip the PDB check at all (as this is something explicitly set by user and not just some CA heuristic), so can you just remove that part of the comment?